libstratum/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@ set(SOURCES
 )
 
 add_library(ethstratum ${SOURCES})
-target_link_libraries(ethstratum PUBLIC devcore Boost::system jsoncpp_lib_static)
+target_link_libraries(ethstratum PUBLIC devcore Boost::system jsoncpp)
```

Comment:
This will not work in Hunter builds, because jsoncpp export its libraries by these weird names like `jsoncpp_lib_static`, `jsoncpp_lib_shared`.